apis/fluentbit/v1alpha2/plugins/output/http_types.go

```diff
@@ -57,6 +57,8 @@ type HTTP struct {
 	GelfFullMessageKey string `json:"gelfFullMessageKey,omitempty"`
 	// Specify the key to use for the level in gelf format
 	GelfLevelKey string `json:"gelfLevelKey,omitempty"`
+	// Integer value to set the maximum number of retries allowed. N must be >= 1 (default: 2)
+	RetryLimit *int32 `json:"retryLimit,omitempty"`
```

Comment:
The `retryLimit` can be set independently on each output section, so we suggest to add it to [common](https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/common_types.go#L7).

And the type of `retryLimit` should not be `int32`, because the values can also be `no_limits `,`false`, and `no_retries`.

Reply:
oak, got it I hope the new version I pushed should achieve that.